src/level/robotmain.cpp

```diff
@@ -1444,6 +1471,8 @@ void CRobotMain::ExecuteCmd(const std::string& cmd)
 
     if (m_phase == PHASE_SIMUL)
         m_displayText->DisplayError(ERR_CMD, Math::Vector(0.0f,0.0f,0.0f));
+
+    m_isCommand = false;
```

Comment:
Pleeease don't introduce a global variable to `CRobotMain` for something that can be local to a function, it's already a big mess. There is a ton of better ways to handle this. For example you can just return it as return value from `ExecuteCmd`.

Actually, I personally don't think this is needed at all, keeping invalid commands in the history is a good way to fix accidental typos without having to retype the whole command.